lib/identity/client.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# @abstract Client for querying Penn State's identity API: https://identity.apps.psu.edu/search-service/resources
+module Identity
```

Comment:
The service, as named by EIT, is "search-service" which is also equally vague. _shrugs_

Reply:
We have `PsuDir` as an example: https://github.com/psu-libraries/psu_dir/blob/master/lib/psu_dir.rb#L10
Which I'm not crazy about. Since this is going to be a gem, maybe `PsuSearchService`? or `PSU::SearchService`? It seems appropriate to have a top-level Penn State module namespace for Ruby/Rails, even though there won't necessarily be anything in it except for gems that use the namespace.

Reply:
I dig the `PSU` top level namespace. There may be some other things we can put in there too... like... something to do with OAuth is rattling around in my brain but I can't quite catch it.